packages/blade/src/components/Button/BaseButton/StyledBaseButton.native.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components/native';
+import getBaseButtonStyles from './getBaseButtonStyles';
+import type { StyledBaseButtonProps } from './StyledBaseButton';
+
+const StyledPressable = styled.Pressable(
+  ({
+    activeBorderColor,
+    activeColor,
+    borderColor,
+    minHeight,
+    spacing,
+    color,
+    disabled,
+    focusBorderColor,
+    focusColor,
+    focusRingColor,
+    hoverBorderColor,
+    hoverColor,
+    isFullWidth,
+    borderWidth,
+    borderRadius,
+  }: Omit<StyledBaseButtonProps, 'children' | 'onClick'>) =>
```

Comment:
You are anyway destructuring the props inside getBaseButtonStyles and not using any ...spreading right? then we can directly do this 

```ts

const StyledPressable = styled.Pressable(
  (props: Omit<StyledBaseButtonProps, 'children' | 'onClick'>) => getBaseButtonStyles(props),
);
```

Reply:
i think it's fine to be explicit, since if there's any bug that might be introduced in BaseButton and any dev on DS passes any prop which shouldn't be passed then it might be bypassed if we add `props`.

Reply:
Yeah agree about being explicit, the point was that we are already being explicit inside the getBaseButtonStyles function. So seems like double work to explicitly destructure props twice.

Reply:
We're being explicit with `getBaseButtonStlyes`. It makes sense to not destruct it here and use `props`.